app/views/publishers/_home_balances.html.slim

```diff
@@ -15,10 +15,13 @@
         - if payout_in_progress?(@publisher)
           .payout_in_progress
             span
-              - if converted_balance.present? && converted_balance.to_i>= PayoutReport::MINIMUM_BALANCE_AMOUNT
-                i = t(".payout_in_progress")
+              - if @publisher.is_selected_wallet_provider_uphold?
+                - if qualifies_for_payout?(@publisher)
+                  = t(".payout_in_progress_uphold", amount: PayoutReport::MINIMUM_BALANCE_AMOUNT)
+                - else
+                  = t(".payout_minimum_balance", amount: PayoutReport::MINIMUM_BALANCE_AMOUNT)
               - else
-                i = t(".payout_minimum_balance", amount: PayoutReport::MINIMUM_BALANCE_AMOUNT, currency: t("bat_in_locale"))
+                = t(".payout_in_progress")
```

Comment:
You should only show if they have a balance

Reply:
Done.